include/systems/game_system.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef FILE_GAME_SYSTEM
+#define FILE_GAME_SYSTEM
+
+#include <memory>
+#include "entities/entity_factory.hpp"
+#include "brickengine/systems/system.hpp"
+#include "controllers/game_controller.hpp"
+
+#include "brickengine/components/player_component.hpp"
+#include "components/health_component.hpp"
+
+class GameSystem : public System {
+public:
+    GameSystem(std::shared_ptr<EntityManager> em, GameController& gc);
+    void update(double deltatime);
+private:
+    static constexpr int MAX_LEVELS = 4;
```

Comment:
~This is 10 by default~
~What is this int for?~
After reading the rest of the changes I think this should be 10. This is the default amount of levels, right?

Reply:
Whoops, did not mean to commit this.